sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/models/BlobsSetImmutabilityPolicyHeaders.java

```diff
@@ -0,0 +1,188 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.storage.blob.implementation.models;
+
+import com.azure.core.annotation.Fluent;
+import com.azure.core.util.DateTimeRfc1123;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
+import java.time.OffsetDateTime;
+
+/** The BlobsSetImmutabilityPolicyHeaders model. */
+@JacksonXmlRootElement(localName = "null")
```

Comment:
@srnagar, @jianghaolu do we know if this will cause any problems in serialization/deserialization? Is this going to expect this class to be wrapped in null:

```xml
<null>
  
</null>
```